gce/test/DeployTest.php

```diff
@@ -0,0 +1,105 @@
+<?php
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\GettingStarted;
+
+use Google\Cloud\TestUtils\FileUtil;
+use Google\Cloud\TestUtils\DeploymentTrait;
+use Google\Cloud\TestUtils\EventuallyConsistentTestTrait;
+use Google\Cloud\TestUtils\TestTrait;
+use Google\Cloud\Utils\ExponentialBackoff;
+use Google\Cloud\Logging\LoggingClient;
```

Comment:
If the fluentd agent is present, what is the logging client needed for?

Reply:
This is unused. Removed!